src/Core.UnitTests/Binding/ServerConnectionTests.cs

```diff
@@ -40,31 +41,33 @@ public void Ctor_SonarCloud_NullOrganization_Throws()
     [TestMethod]
     public void Ctor_SonarCloud_NullSettings_SetDefault()
     {
-        var sonarCloud = new ServerConnection.SonarCloud(org, null);
+        var sonarCloud = new ServerConnection.SonarCloud(Org, null);
 
         sonarCloud.Settings.Should().BeSameAs(ServerConnection.DefaultSettings);
     }
     
     [TestMethod]
     public void Ctor_SonarCloud_NullCredentials_SetsNull()
     {
-        var sonarCloud = new ServerConnection.SonarCloud(org, credentials: null);
+        var sonarCloud = new ServerConnection.SonarCloud(Org, credentials: null);
 
         sonarCloud.Credentials.Should().BeNull();
     }
     
     [TestMethod]
     public void Ctor_SonarCloud_SetsProperties()
     {
+        var expectedServerUri = new Uri("https://sonarcloud.io");
         var serverConnectionSettings = new ServerConnectionSettings(false);
         var credentials = Substitute.For<ICredentials>();
-        var sonarCloud = new ServerConnection.SonarCloud(org, serverConnectionSettings, credentials);
+        var sonarCloud = new ServerConnection.SonarCloud(Org, serverConnectionSettings, credentials);
 
-        sonarCloud.Id.Should().BeSameAs(org);
-        sonarCloud.OrganizationKey.Should().BeSameAs(org);
+        sonarCloud.Id.Should().BeSameAs(Org);
+        sonarCloud.OrganizationKey.Should().BeSameAs(Org);
         sonarCloud.ServerUri.Should().Be(new Uri("https://sonarcloud.io"));
         sonarCloud.Settings.Should().BeSameAs(serverConnectionSettings);
         sonarCloud.Credentials.Should().BeSameAs(credentials);
+        sonarCloud.CredentialsUri.Should().Be(new Uri(expectedServerUri, $"{SonarCloud.Organizations}/{sonarCloud.OrganizationKey}"));
```

Comment:
this feels like a copy paste of the implementation. using a hardcoded string here could be safer